src/DataFramesMeta.jl

```diff
@@ -143,6 +148,10 @@ function replace_dotted!(e, membernames)
     Expr(:., x_new, y_new)
 end
 
+getsinglecolumn(df, s::DataFrames.ColumnIndex) = df[!, s]
```

Comment:
@bkamins This is what I mean when I say that we deprecate `cols([:x, :y])` in side `@with`. 

I define a small helper function for getting a single column from the data frame. Below, I use this function when getting columns to the anonymous function created.